Snippets/ServiceControl/ServiceControl_All/Core_[,6.0]/ErrorQueueMsmqConfigurationProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿#pragma warning disable 618
```

Comment:
should that be v6 inclusive or exclusive? hard to figure out which versions it applies to
also, is anything below v5 supported by servicecontrol?

@WilliamBZA ☝️